openedx/core/djangoapps/content/learning_sequences/api/processors/milestones.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from django.contrib.auth import get_user_model
+from opaque_keys.edx.keys import CourseKey
+from util import milestones_helpers
+
+from .base import OutlineProcessor
+
+User = get_user_model()
+log = logging.getLogger(__name__)
+
+
+class MilestonesOutlineProcessor(OutlineProcessor):
+    """
+    Responsible for applying all milestones outline processing.
+    """
+    def inaccessible_sequences(self, full_course_outline):
+        """
+        Returns the set of sequence usage keys for which the
+        user has pending milestones
+        """
+        inaccessible = set()
```

Comment:
We should also return `set()` if the user is staff or are we checking that somewhere else? Same for the `ContentGatingProcessor`.

Reply:
If I am not mistaken, all processors are ignored for staff

Reply:
Yeah. The processors do run in that they `load_data`, but neither `inaccessible_sequences` nor `usage_keys_to_remove` applies to staff, so they're skipped entirely.